site/docs/master/release.md

```diff
@@ -0,0 +1,36 @@
+This file documents the list of steps to perform to create a new Antrea
+release. We use `<TAG>` as a placeholder for the release tag (e.g. `v0.1.0`).
+
+ * Create a release branch for each new minor release (e.g `release-0.1`). For
+   bug fixes, use the appropriate release branch.
+
+ * Open a PR against the appropriate release branch with the following commits:
+    1. a commit to update the [CHANGELOG]({{site.gh_repo}}/tree/master/CHANGELOG.md).
```

Comment:
@ccbayer this won't work; because the file is symbolically linked from the root .